packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1656,17 +1765,33 @@ export class Combobox
   }
 
   private renderListBoxOptions(): JsxNode {
-    return this.filteredItems.map((item) => (
-      <li
-        ariaLabel={item.label}
-        ariaSelected={item.selected}
-        id={item.guid ? `${itemUidPrefix}${item.guid}` : null}
-        role="option"
-        tabIndex="-1"
-      >
-        {item.heading || item.textLabel}
-      </li>
-    ));
+    const selectAllComboboxItem = this.selectAllEnabled &&
+      this.selectionMode !== "single" &&
+      this.selectionMode !== "single-persist" && (
+        <calcite-combobox-item
```

Comment:
Shouldn't this be a native `li` element? This is just for a11y purposes right? Do we even need to have the select all option here? It seems like we don't need to have this here.

Reply:
I think the hidden accessibility list should mirror the visible dropdown, I'd expect that when navigating the combobox with arrow keys, the screen reader needs to announce the "Select All" option just like any other item.

Does this sound right, @geospatialem?

But yeah it should be an `<li>` element.

Reply:
This sounds great with JAWS and provides context. This seems like good context to users with the functionality via the `<li>` element, no additional change needed. 👍🏻